### ISSUE TITLE
Some terms have term_modified dates that don't correspond to the most recent version issued date

### ISSUE THREAD

**Issue by baskaufs** [tdwg/rs.tdwg.org#1]
For example, the curatorial terms apparently only ever had one version issued on 2007-04-17.  However, they have term_modified dates of 2009-04-24.  This is throwing off my html script, which infers the URL of the most recent version from the modified date.  

I think this may be happening because for dead terms, the term_modified date is the date of deprecation, which doesn't actually result in the generation of a new term version - rather it changes the status of the most recent version.

Basically, this is a problem relating to documenting the date of deprecation, which isn't really captured in a specific property.

**Comment by baskaufs** [comment]
I don't remember how I fixed this problem but it doesn't seem to be causing any problems with generating the web pages. There is still the general issue of how the date of deprecation is captured. If the deprecated term is replaced by another term, then one could probably infer the deprecation date from the issuing date of the replacing term. This is the case for terms like http://rs.tdwg.org/dwc/curatorial/Disposition which has a last modified date of 2009-04-24 and is replaced by http://rs.tdwg.org/dwc/terms/disposition.ttl which has a creation date of 2009-04-24.
